application/common/components/MfaBackendInterface.php

```diff
@@ -35,4 +35,16 @@ public function verify(int $mfaId, string $value, string $rpOrigin = '');
      * @return bool
      */
     public function delete(int $mfaId): bool;
+
+
+
+    /**
+     * Delete WebAuthn credential
```

Comment:
Not sure if this comment should be changed or not.

```suggestion
     * Delete credential
```